.rubocop.yml

```diff
@@ -20,7 +20,7 @@ GraphQL/ObjectDescription:
     - app/graphql/types/query_type.rb
 
 RSpec/NestedGroups:
-  Max: 5
+  Max: 4
```

Comment:
Just wondering the reason for the reduction here?

Reply:
We increased it from 4->5 in [the commit which added the scratch-only option](https://github.com/RaspberryPiFoundation/editor-api/commit/56022c6f1dbea4ea1f94c95497aab3971e7c3d2d), so it seemed right to revert that change since the higher limit isn't needed anywhere else.